src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -83,10 +96,13 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
   @Output()
   public inlineFormClose = new EventEmitter<SkyInlineFormCloseArgs>();
 
-  public contentId: string = `sky-radio-content-${++nextContentId}`;
+  public contentId: string = `sky-repeater-item-content-${++nextContentId}`;
```

Comment:
Good find, but could this be considered a breaking change if a consumer is using `ids` to grab something?

Reply:
In my opinion - we shouldn't consider this a breaking change. This was a blatant bug to where we were giving a completely incorrect ID. We have set the precedence in the past to go ahead and fix things like that since it was something that was obviously a bug to begin with and not a feature that we are changing.

Reply:
Didn't know we had a precedent for it. Good to know 👍